server/controllers/methods.js

```diff
@@ -0,0 +1,9 @@
+const db = require('../dbconnection/schema.js'); //accese to your scheema;
+
+exports.addData = (req,res) => {
+	db.create(req.body).then(results => res.send(results))
```

Comment:
requires  a .catch function to handle errors

Reply:
a post request does not require any data to be sent back as a response

Reply:
catch for error should be there, thanks will add that in and yes you are right shouldn't have to send back anything.